lasagne/nonlinearities.py

```diff
@@ -9,12 +9,10 @@
 # sigmoid
 def sigmoid(x):
     """Sigmoid activation function :math:`\\varphi(x) = \\frac{1}{1 + e^{-x}}`
-
```

Comment:
In commit https://github.com/Lasagne/Lasagne/pull/843/commits/58904e3d9de56c4a0de068addc72ac96429bdb6f, you removed almost all blank lines in lasagne/nonlinearities.py. Please undo!